### ISSUE TITLE
Search engine optimization

### ISSUE THREAD

**Issue by NickRoach** [Blue-Mountains-Soaring-Club/website#9]
When I search for "blue mountains soaring club" with Google, with or without the quotes, it is not found! This is a big problem because anyone who is not already connected with the club will have a very hard time finding the site, and it's also a cognitive strain for current members to remember the address. This should be fixed.

This is done when: 

A user searches "blue mountains soaring club", with or without quotes, and the website is the first result
A user searches blue mountains paragliding (without quotes) and this website is on the first page of results
A user searches blue mountains hang gliding (without quotes) and this website is on the first page of results

**Comment by absoludity** [comment]
Absolutely - thanks Nick.

**Comment by NickRoach** [comment]
I am happy to do this, and I will, but I am not able to assign myself on github

**Comment by absoludity** [comment]
> I am happy to do this, and I will, but I am not able to assign myself on github

Try now? I just added you as a maintainer. Thanks Nick!

**Comment by NickRoach** [comment]
@absoludity I think I have identified the problem. Google Search Console points out that we have <meta name="robots" content="noindex, nofollow"> in the pages. That has come from the theme. Looking at the theme repo readme, I find that we need to set HUGO_ENV=production hugo for the build command
https://github.com/theNewDynamic/gohugo-theme-ananke/blob/master/README.md#production:~:text=To%20run%20in,HUGO_ENV%3Dproduction%20hugo
The build runs on Netify though, and I don't have access to that. Can you please edit the build script or give me access to Netify so that I can do it?

**Comment by NickRoach** [comment]
Great, thanks @absoludity. I have requested Google to index the site and it's doing it now. I expect that the site will appear in search results pretty soon.
Netify seems to work fine and I don't think the deployment will need to be modified very often, so I'm happy to leave things as they are and not to have access to it. When we're not being paid there is no point creating extra work ;)

**Comment by absoludity** [comment]
This can be closed now, can it, @NickRoach ? Or is there something else you wanted to do.

**Comment by NickRoach** [comment]
Yes! I had left it open because those last two criteria hadn't been met, but I see that Google's algorithm has now learned that this website is the thing people are looking for. We can tick them off and close

**Comment by absoludity** [comment]
Number one on both google:

![Image](https://github.com/user-attachments/assets/c61e6314-3c1e-489f-aac4-aa05b3d4f013)

and duckduckgo:

![Image](https://github.com/user-attachments/assets/8ce7e1d2-e34a-4a0f-b795-94199789e71f)

even with other terms, like "paraglide blue mountains"

![Image](https://github.com/user-attachments/assets/fafc3fa9-3284-495e-a382-62a2bc3fa2eb)

Nice!
